src/components/Semester.vue

```diff
@@ -319,21 +322,21 @@ export default {
       const _this = this;
       for (let i = 0; i < splitReq.length; i++) {
         if (splitReq[i].indexOf('"') >= 0) {
-          splitReq[i] = 'true';
-        }
-        if ('()/, '.indexOf(splitReq[i]) < 0) {
+          //don't include things like 'permission of instructor'
+          splitReq[i] = '';
+          //remove operator before & parentheses if directly before and after
+          if(splitReq[i] !== '(') {
+            splitReq[i-1] = '';
```

Comment:
This feels a little dangerous, and I haven't come across an example (in a short search) where it's not already an empty string. Why do we need this particular case?

Reply:
Unfortunately it's been a while and I don't 100% understand my comments either but I think what this is doing is 
1)  if the part right before it isn't an open parenthesis it must be an operator (',' or '/'), so that should be removed (set to '') since the string is being removed
2) if there are parenthesis before and after, remove both of them

An example I made would be "1.01/2.003/(4.05,6.06,7.07,\"permission of instructor\")"
In this case there is a comma before "permission of instructor" and a parenthesis after.  The comma and string should be removed so it becomes "1.01/2.003/(4.05,6.06,7.07)"

Reply:
ah I see now, I didn't get what was happening before. thanks